README.md

```diff
@@ -2,7 +2,7 @@
 
 # Dora-e-Quran Notes
 
-This repo contains the notes devised by me after listening to Dr. Israr Sahab's Dora-e-Quran Videos.
+This repo contains the notes devised by contributors after listening to Dr. Israr Sahab's Dora-e-Quran videos.
```

Comment:
by me and the [contributors](https://github.com/miankhalid/dora-e-quran/graphs/contributors)